docs/_v15_x/2_send.md

```diff
@@ -267,7 +255,86 @@ client.createMessage(message)
         .send();
 ```
 
+## Send messaage with request for registration
+
+It is possible to send a message to a person, who does not have a Digipost account, where the message triggers an SMS notification with a request for registration. The SMS notification says that if they register for a Digipost account the document will be delivered digitally. If the user does not register for a Digipost account within the defined deadline, the document will either be delivered as physical mail or not at all.
+
+### Request for registration with physical mail as fallback
+
+In this case the document will be delivered as physical mail if the recipient has not registered for a Digipost account by the defined deadline.
+
+```java
+UUID documentId = UUID.randomUUID(); 
+Document document = new Document(documentId, "Hello!", FileType.PDF); 
+
+PrintDetails printDetails = new PrintDetails(RECIPIENT, RETURN_RECIPIENT); 
+ 
+RequestForRegistration requestForRegistration = new RequestForRegistration(
+// Deadline for when the recipent can no longer register a Digipost account
+    ZonedDateTime.now().plus(6, ChronoUnit.HOURS),
+// Phone number that will be used for the SMS notification
+    new PhoneNumber("+4712345678"), 
+    null, 
+    printDetails 
+); 
+ 
+UUID messageId = UUID.randomUUID(); 
+Message message = Message.newMessage(messageId.toString(), document) 
+    .recipient(new PersonalIdentificationNumber("12345678901")) 
+    .senderId(senderId) 
+    .requestForRegistration(requestForRegistration) 
+    .build(); 
+ 
+MessageDelivery delivery = sendClient.createMessage(message) 
+    .addContent(document, Contents.filFraDisk("gyldig-for-print.pdf")) 
+    .send(); 
+
+System.out.println("status: " + delivery.getStatus()); 
+System.out.println("channel: " + delivery.getChannel()); 
+```
+
```

Comment:
```suggestion

If the recipient does not have a Digipost account already, the value of `getChannel()` will be `null`, otherwise `Channel.DIGIPOST`.

```